test/functional/feature_xfield_maxblocksize.py

```diff
@@ -494,7 +494,7 @@ def run_test(self):
         self.block_time += 1
         blocknew = self.new_block(48, spend=self.unspent[21])
         blocknew.solve(self.aggprivkey[1])
-        node.p2p.send_blocks_and_test([blocknew], node, success=True, request_block=False, timeout=TAPYRUSD_REORG_TIMEOUT)
+        node.p2p.send_message(msg_block(blocknew))
```

Comment:
Why is this change necessary for cross-compiling.

Reply:
This change is not related to the cross compilation.  In some of the Ci jobs this is the exact line where the timeout occurred and the job fails. This statement results in a blockchain reorg. So I changed it in such a way that there is no time keeping in the same statement. Now we just send the block. Then wait for the node to reorganize and reach the correct tip in the subsequent statements.